### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by adamj88**
Added composer.json file so the extension can be installed easily via composer.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "rk/typecms",
```
Comment (rskuipers) [inline composer.json]: Could you change this to rskuipers? :)

**Comment by rskuipers** [comment]
Thanks for the contribution! Had just one remark and will merge after that's fixed :)

**Comment by adamj88** [comment]
@rskuipers no problem :) looks like a great idea to improve one of Magento's weak spots, the CMS. Wish I'd found it sooner!

**Comment by rskuipers** [comment]
@adamj88 Thanks! I did everything I could to draw attention to it but there didn't seem to be a need.

Merged! :+1:

**Comment by rskuipers** [comment]
@adamj88 https://packagist.org/packages/rskuipers/typecms

**Pull request merged**